gradle.properties

```diff
@@ -17,8 +17,8 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-PROJECT_VERSION = 3.10.0
-PROJECT_GROUP = com.graphql-java
+PROJECT_VERSION = 3.10.0-SNAPSHOT
+PROJECT_GROUP = com.smartling
```

Comment:
I made mine `com.smartling.graphql-java` - should we try to stay consistent?

Reply:
fixed